core/src/test/java/org/frankframework/jdbc/FixedQuerySenderTest.java

```diff
@@ -41,18 +42,18 @@ public class FixedQuerySenderTest {
 
 	private PipeLineSession session;
 
-	private String dataSourceName;
+	private Dbms databaseUnderTest;
 
 	@BeforeEach
 	public void setup(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
-		this.dataSourceName = databaseTestEnvironment.getDataSourceName();
+		this.databaseUnderTest = databaseTestEnvironment.getDbmsSupport().getDbms();
 		TestConfiguration configuration = databaseTestEnvironment.getConfiguration();
 		session = new PipeLineSession();
 		session.put(PipeLineSession.MESSAGE_ID_KEY, ConfiguredTestBase.testMessageId);
 		session.put(PipeLineSession.CORRELATION_ID_KEY, ConfiguredTestBase.testCorrelationId);
 
 		fixedQuerySender = new FixedQuerySender();
-		fixedQuerySender.setDatasourceName(dataSourceName);
+		fixedQuerySender.setDatasourceName(databaseUnderTest.getProductName());
```

Comment:
Is product name hier wel de juiste waarde?

Reply:
Hm je hebt gelijk dat ik hier beter `databaseTestEnvironment.getDataSourceName()` had kunnen gebruiken. Raar dat ik dat gedaan heb.